simulator/simulator.go

```diff
@@ -65,10 +57,20 @@ func startSimulator() error {
 		return xerrors.Errorf("create an etcd client: %w", err)
 	}
 
-	// need to sleep here to make all controllers create initial resources. (like "system-" priorityclass.)
-	time.Sleep(1 * time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	defer cancel()
+	wait.UntilWithContext(ctx, func(ctx context.Context) {
```

Comment:
Sorry, I misled in a previous comment.
We should use PollUntilContextTimeout rather, otherwise `wait.UntilWithContext` wouldn't panic or error out if reaching timeout seemingly.
https://pkg.go.dev/k8s.io/apimachinery/pkg/util/wait#PollUntilContextTimeout